week2/address_book/addressbook_create_content/7_event_handling.md

```diff
@@ -0,0 +1,64 @@
+# Event handling
+
+Right now, when we click on the save button, nothing happens. The reason for this is because we don't have any action registered to execute when the button is clicked, let's change that.
+
+Add the following code in the `src/app.js` file. You can remove any code you have there at the moment.
+
+
+```javascript
+document.addEventListener('DOMContentLoaded', () => {
+  // Select form object from the DOM
+  const addContactForm = document.querySelector('.new-contact-form')
+
+  // Register an event to listen for form submission
+  addContactForm.addEventListener('submit', event => {
+    // Disable default behavior when submitting form
+    event.preventDefault()
+
+    // Get all inputs elements from the form
+    const {
+      name,
+      email,
+      phone,
+      company,
+      notes,
+      twitter,
+    } = addContactForm.elements
+
+    // Create contact object
+    const contact = {
+      id: Date.now(),
+      name: name.value,
+      email: email.value,
+      phone: phone.value,
+      company: company.value,
+      notes: notes.value,
+      twitter: twitter.value,
+    }
+
+    console.log(`Saving the following contact: ${JSON.stringify(contact)}`)
+  })
+})
+```
+
+Let's go over what we're doing here.
+
+Before we can interact with anything on our page \(i.e clicking on buttons, filling forms, etc...\) we need to make sure that they're present on the DOM. This is what we use the `DOMContentLoaded` event for.
+
+We add and event listener to the document to watch for `DOMContentLoaded` event. When the page loads and every element it drawn on the page, it will fire up and execute the callback function we've defined.
```

Comment:
We add and event listener - and should be an
...and every element it drawn - it should be is